src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -145,6 +157,24 @@ def _get_current_exposure(self, index: pd.Index) -> pd.Series:
             if mam_coverage == "baseline":  # return standard exposure if baseline
                 propensity = self.propensity(index)
                 return pd.Series(self.exposure_distribution.ppf(propensity), index=index)
+            elif mam_coverage == "targeted":
+                # initialize exposures as 0s using index
+                exposures = pd.Series(0, index=index)
+
+                # define relevant booleans
+                wasting = self.wasting_exposure(index)
+                age = self.population_view.get(index)["age"]
+                underweight = self.underweight_exposure(index)
+
+                in_mam_state = wasting == "cat2"
+                in_age_range = (age >= 0.5) & (age < 2)
+                is_severely_underweight = underweight == "cat3"
+
+                is_covered = (in_mam_state & in_age_range) | (
+                    in_mam_state & is_severely_underweight
+                )
```

Comment:
I looked at the docs and they say this is the population we should be covering:

> In modeling, the targeted MAM intervention will be given to children who: 
>
> - Fall in the MAM category for wasting (WHZ z-score < -2 to -3) AND
> - Are aged 6-24 months OR are underweight with WAZ z-score < -3

This boolean expression reads ambiguously to me. Can you get confirmation from Ali that what you've implemented is what she intends (and maybe ask her to make the docs a bit clearer)?